packages/nimble-components/src/table/tests/table-grouping.spec.ts

```diff
@@ -0,0 +1,625 @@
+import { html } from '@microsoft/fast-element';
+import type { Table } from '..';
+import type { TableColumnText } from '../../table-column/text';
+import { waitForUpdatesAsync } from '../../testing/async-helpers';
+import { type Fixture, fixture } from '../../utilities/tests/fixture';
+import { TableColumnSortDirection, TableRecord } from '../types';
+import { TablePageObject } from './table.pageobject';
+
+interface SimpleTableRecord extends TableRecord {
+    id: string;
+    stringData1?: string | null;
+    stringData2?: string | null;
+    stringData3?: string | null;
+}
+
+// prettier-ignore
+async function setup(): Promise<Fixture<Table<SimpleTableRecord>>> {
```

Comment:
I might have missed it, but I didn't see a test that validates that you can sort by a hidden column. If that scenario isn't covered by an existing test, it would be good to add a test for it.